Tasks/Denkevich/Task6/Task6.py

```diff
@@ -0,0 +1,90 @@
+# My realizations (using only my knowledge)
+
+# Implementation 1:
+l = [i for i in range(1, 101)]
+la = []
+for i in l:
+    if i % 3 == 0 and i % 5 == 0:
+        la.append('FizzBuzz')
+    elif i % 3 == 0:
+        la.append('Fizz')
+    elif i % 5 == 0:
+        la.append('Buzz')
+    else:
+        la.append(f'{i}')
+s = ', '.join(la)
+# print(s)
+print("-----------------------")
+
+# Implementation 2:
+print(", ".join([str(i) if i % 3 != 0 and i % 5 != 0 else 'FizzBuzz' if i % 3 == 0 and i % 5 == 0 else 'Fizz' if i % 3 == 0 else 'Buzz' for i in range(1, 101)]))
+print("-----------------------")
+
+# Implementation 3:
+import time
+def FizzBuzz(s,i):
+    n = 1
+    while n <= i:
+        time.sleep(0.2)
+        print(s if n % 3 == 0 and n % 5 == 0 else s[0:4] if n % 3 == 0 else s[4:] if n % 5 == 0 else n)
+        n += 1
+FizzBuzz('FizzBuzz', 100)
+
+# Implementation 4:
+def FizzBuzz():
+    s = 'FizzBuzz'
+    print([i if i % 3 != 0 and i % 5 != 0 else s if i % 3 == 0  and i % 5 == 0 else s[0:4] if i % 3 == 0 else s[4:] for i in range(1,101)])
+FizzBuzz()
+
+# Implementation 5:
+def FizzBuzz(f, b):
+    print([f"{f}{b}" if i % 3 == 0 and i % 5 == 0 else f if i % 3 == 0 else b if i % 5 == 0 else i for i in range(1, 101)])
+FizzBuzz('Fizz', 'Buzz')
+
+# Implementation found in the internet:
+# Implementation 6:
+s = ''
+for i in range(1, 101):
+    if i % 3 == 0:
+        s += 'Fizz'
+    if i % 5 == 0:
+        s += 'Buzz'
+    if i % 3 != 0 and i % 5 != 0:
+        s += str(f" {i} ")
+print(s)
+
+# Implementation 7:
+l = []
+for i in range(1, 101):
+    if i % 15 == 0:
+        l.append('FizzBuzz')
+    elif i%3 == 0:
+        l.append('Fizz')
+    elif i%5 == 0:
+        l.append('Buzz')
+    else:
+        l.append(i)
+print(l)
+
+# And Implementation 8.
+# The most butiful realization I found!
+print(*('FizzBuzz'[(0,4,4)[x % 3]:4 if x % 5 else 8] or x for x in range(1,101)))
```

Comment:
this one is totally overcomplicated, there is a simplest version based on the same approach with the list comprehension